app/src/main/java/com/yeonae/chamelezone/data/source/remote/course/CourseRemoteDataSourceImpl.kt

```diff
@@ -18,26 +21,50 @@ class CourseRemoteDataSourceImpl private constructor(private val courseApi: Cour
         placeNumber: Int,
         title: String,
         content: String,
+        image: String,
         callBack: CourseCallBack<String>
     ) {
-        val jsonObject = JsonObject().apply {
-            addProperty("memberNumber", memberNumber)
-            addProperty("placeNumber", placeNumber)
-            addProperty("title", title)
-            addProperty("content", content)
+        val image =
+            MultipartBody.Part.createFormData(
+                "images",
+                image,
+                RequestBody.create(MediaType.parse("image/*"), File(image))
+            )
+
+        val memberNumber = RequestBody.create(
+            MediaType.parse("text/plain"), memberNumber.toString()
+        )
+        var placeNumber = ArrayList<RequestBody>()
+        for (i in placeNumber.indices) {
```

Comment:
```suggestion
        for (number in placeNumber) {  혹은 placeNumber.forEach { number ->
```
i 라는 인덱스 참조 변수를 굳이 만들필요 없는 상황으로 보입니다.

Reply:
수정했습니다